Assets/MRTK/SDK/Experimental/Features/UX/BoundsControl/Visuals/Configuration/PerAxisHandlesConfiguration.cs

```diff
@@ -0,0 +1,111 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Experimental.UI.BoundsControlTypes;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.UI.BoundsControl
+{
+    /// <summary>
+    /// Configuration for <see cref="PerAxisHandles"/> used in <see cref="BoundsControl"/>.
+    /// This class provides all data members needed to create axis based handles for <see cref="BoundsControl"/>.
+    /// </summary>
+    public abstract class PerAxisHandlesConfiguration : HandlesBaseConfiguration
+    {
+
+        [SerializeField]
+        [Tooltip("Determines the type of collider that will surround the handle prefab.")]
+        private HandlePrefabCollider handlePrefabColliderType = HandlePrefabCollider.Box;
+
+        /// <summary>
+        /// Determines the type of collider that will surround the handle prefab.
+        /// </summary>
+        public HandlePrefabCollider HandlePrefabColliderType
+        {
+            get
+            {
+                return handlePrefabColliderType;
+            }
+            set
+            {
+                if (handlePrefabColliderType != value)
+                {
+                    handlePrefabColliderType = value;
+                    colliderTypeChanged.Invoke();
+                }
+            }
+        }
+
+        [SerializeField]
+        [Tooltip("Check to show handles for the X axis.")]
+        private bool showHandleForX = true;
+
+        /// <summary>
+        /// Check to show handles for the X axis.
+        /// </summary>
+        public bool ShowHandleForX
+        {
+            get
+            {
+                return showHandleForX;
+            }
+            set
+            {
+                if (showHandleForX != value)
+                {
+                    showHandleForX = value;
+                    handlesChanged.Invoke(HandlesChangedEventType.Visibility);
+                }
+            }
+        }
+
+        [SerializeField]
+        [Tooltip("Check to show handles for the Y axis.")]
+        private bool showHandleForY = true;
+
+        /// <summary>
+        /// Check to show handles for the Y axis.
```

Comment:
Nit: The comments and the tooltips are the same but it is a bit weird to see "Check to show handles" on a boolean property right